In [ ]:
import pandas as pd
import numpy as np
from scipy.stats import chi2_contingency
import seaborn as sns

In [ ]:
df = pd.read_csv('/content/BuyerRatio.csv')

In [ ]:
df.head()

Observed Values  East  West  North  South
0           Males    50   142    131     70
1         Females   435  1523   1356    750

In [ ]:
contigency = pd.crosstab(df["Observed Values"], df)

In [ ]:
df = df.drop(['Observed Values'], axis =1)

In [ ]:
df

East  West  North  South
0    50   142    131     70
1   435  1523   1356    750

In [ ]:
stat, p, dof, expected = chi2_contingency(df)

In [ ]:
expected

array([[  42.76531299,  146.81287862,  131.11756787,   72.30424052],
       [ 442.23468701, 1518.18712138, 1355.88243213,  747.69575948]])

In [ ]:
p

0.6603094907091882

#Since the Pvalue (0.66) > alpha(0.050), we failed to reject Ho
#Hence the buying ratio of Male and Female are equal accross the regions
